resources/com.sap.piper/pipeline/cloudSdkToGppConfigSettings.yml

```diff
@@ -37,6 +37,12 @@ removedOrReplacedConfigKeys:
     postAction: true
     customMessage: "Details can be found in the release notes of v43 of the Cloud SDK Pipeline: https://github.com/SAP/cloud-s4-sdk-pipeline/releases/tag/v43"
 
+requiredConfigKeys:
+  inferBuildTool:
```

Comment:
what if you've set `buildTool` directly? This would make the pipeline fail, right? do we want that?

Reply:
At the moment, inferBuildTool does more than setting buildTool. The point where I got stuck in testing was that artifactPrepareVersion is only used then. So it seemed like a requirement to me for the time being that inferBuildTool must be configured, or else other changes would be required.